massa-network-exports/src/commands.rs

```diff
@@ -175,9 +176,9 @@ pub enum NetworkCommand {
         header: WrappedHeader,
     },
     /// `(PeerInfo, Vec <(NodeId, bool)>) peer info + list` of associated Id nodes in connection out (true)
-    GetPeers(oneshot::Sender<Peers>),
+    GetPeers(Sender<Peers>),
```

Comment:
I chose crossbeam over the std one, because `select` has been removed from the std one.

Reply:
It was tokio ones before not std

Reply:
I thought you meant why I chose `crossbeam` over `std`, I changed to a threaded channel implementation, since the goal of the PR is removing async where possible.